rca/api_content/content.py

```diff
@@ -53,11 +55,14 @@ def try_and_fetch(url):
         error_text = "Error occured when fetching further detail data"
         logger.exception(error_text)
         raise CantPullFromRcaApi(error_text)
```

Comment:
Rather than halting the program, would it be worth returning `None` here?

Reply:
Same for above

Reply:
I'm not sure I know the answer to this - what I can say though is that the `CantPullFromRcaApi` exception is set to pass so it shouldn't halt it?

Reply:
We covered this on a call, the general recommendation is that we refactor the model code so it's not try to catch this exception [here](https://github.com/torchbox/rca-wagtail-2019/blob/master/rca/home/models.py#L186) - I'll roll this into another ticket due to time restraints

Reply:
I'll add it to https://projects.torchbox.com/projects/rca-website-rebuild/tickets/308